### PR TITLE
chore: made prettier our storybook

### PR DESCRIPTION
**Pull request by GutierrezGo**
References:  LOVE ❣️

## Motivation and context

Giving some love to our storybook library

## Before

No sections for the components and no logo.

## After
If you click on the logo, it will send you to the AS24 page in DE

Screenshot 2022-07-15 at 14 41 36

## How to test

Only the theme in storybook has change

### CODE REVIEW

File: src/components/input/size.stories.mdx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import Input from './index.tsx';
 import { Template } from './index.stories.mdx';
 
 <Meta
-  title="Components/Forms/Input/Sizes"
+  title="Forms/Input/Sizes"
```
Comment (Averethel) [inline src/components/input/size.stories.mdx:7]: Stories here are linked in the main input stories, you'd need to change links there (here's where the linked stories are)

File: src/components/input/state.stories.mdx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import Input from './index.tsx';
 import { Template } from './index.stories.mdx';
 
 <Meta
-  title="Components/Forms/Input/States"
+  title="Forms/Input/States"
```
Comment (Averethel) [inline src/components/input/state.stories.mdx:7]: Stories here are linked in the main input stories, you'd need to change links there

File: .storybook/YourTheme.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { create } from '@storybook/theming';
```
Comment (Averethel) [inline .storybook/YourTheme.js:1]: Should we call it `theme` or `as24Theme`? `YourTheme` looks a bit out of place

**Review by Averethel — COMMENTED** [review]
More love is always great 👍🏻  but before I can approve:
- Please make sure that the linked input stories still work in the `docs` tab
- since we only change the storybook configuration and don't introduce any changes to the components themselves I'd go with `chore` to not release a new version

**Review by Averethel — APPROVED** [review]

**Review by Marine-Berthier — APPROVED** [review]
Thank you for the extra love!

**Pull request merged**